CHANGELOG.md

```diff
@@ -24,3 +24,4 @@
 * [ENHANCEMENT] flagext: for cases such as `DeprecatedFlag()` that need a logger, add RegisterFlagsWithLogger. #80
 * [BUGFIX] spanlogger: Support multiple tenant IDs. #59
 * [BUGFIX] Memberlist: fixed corrupted packets when sending compound messages with more than 255 messages or messages bigger than 64KB. #85
+* [ENHANCEMENT] ring/util.go: make WaitRingStability more responsive #96
```

Comment:
This PR's number is 95:
```suggestion
* [ENHANCEMENT] ring/util.go: make WaitRingStability more responsive #95
```

Reply:
mixup, thought it was the issue to be referenced